docs/source/multidc/multidc.md

```diff
@@ -0,0 +1,456 @@
+
+As an example, we walk through the procedure of deploying two datacenters in distinct regions of a selected cloud provider.
+
+``` note::
+    This guide is dedicated to deploying multi-datacenter ScyllaDB clusters and does not discuss unrelated configuration options.
+    For details of ScyllaDB cluster deployments and their configuration, refer to `Deploying Scylla on a Kubernetes Cluster <../generic.md>`_ in ScyllaDB Operator documentation.
+```
+
+## Prerequisites
+
+As this document describes the procedure of deploying a Multi Datacenter ScyllaDB cluster, you are expected to have the required infrastructure prepared.
+We are assuming two interconnected Kubernetes clusters, capable of communicating with each other over PodIPs, with each cluster meeting the following requirements:
+- a node pool dedicated to ScyllaDB nodes composed of at least 3 nodes running in different zones (with unique `topology.kubernetes.io/zone` label), configured to run ScyllaDB, each labeled with `scylla.scylladb.com/node-type: scylla`
+- running ScyllaDB Operator and its prerequisites
+- running a storage provisioner capable of provisioning XFS volumes of StorageClass `scylladb-local-xfs` in each of the nodes dedicated to ScyllaDB instances
+
+You can refer to one of our guides describing the process of preparing such infrastructure:
+- [Build multiple Amazon EKS clusters with Inter-Kubernetes networking](eks.md)
+- [Build multiple GKE clusters with Inter-Kubernetes networking](gke.md)
+
+Additionally, to follow the below guide, you need to install and configure the following tools that you will need to manage Kubernetes resources:
+- kubectl – A command line tool for working with Kubernetes clusters.
+
+See [Install Tools](https://kubernetes.io/docs/tasks/tools/) in Kubernetes documentation for reference.
+
+## Multi Datacenter ScyllaDB Cluster
+
+In v1.11, ScyllaDB Operator introduced support for manual multi-datacenter ScyllaDB cluster deployments.
+
+``` warning::
+    ScyllaDB Operator only supports *manual configuration* of multi-datacenter ScyllaDB clusters.
+    In other words, although ScyllaCluster API exposes the machinery necessary for setting up multi-datacenter ScylaDB clusters, the ScyllaDB Operator only automates operations related to singular datacenters.
+
+    Operations related to multiple datacenters may require manual intervention of a human operator.
+    Most notably, destroying one of the Kubernetes clusters or ScyllaDB datacenters is going to leave DN nodes behind in other datacenters, and their removal has to be carried out manually.
+```
+
+The main mechanism used to set up a manual multi-datacenter ScyllaDB cluster is a field in ScyllaCluster's specification - `externalSeeds`.
+
+### External seeds
+
+The `externalSeeds` field in ScyllaCluster's specification enables control over external seeds propagated to ScyllaDB binary as `seeds` parameter of seed-provider.
```

Comment:
```suggestion
The `externalSeeds` field in ScyllaCluster's specification enables control over external seeds that are propagated to ScyllaDB binary using additional `seeds` arguments.
```

Reply:
seeds-provider is a config options in ScyllaDB config, so this suggestion removes a bit of accuracy - was that intended?

Reply:
i thought we set the `seed` flag, not mess with the config

Reply:
To the docker entrypoint, yes, but that ends up being provided as `--seed-provider-parameters seeds=<seeds>`

Reply:
done